dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/internal/Project.kt

```diff
@@ -0,0 +1,24 @@
+package com.datadog.gradle.plugin.internal
+
+import java.io.ByteArrayOutputStream
+import org.gradle.api.Project
+import org.gradle.process.internal.ExecException
+
+internal fun Project.execShell(vararg command: String): String {
+    val outputStream = ByteArrayOutputStream()
+    val errorStream = ByteArrayOutputStream()
+    try {
+        this.exec {
+            @Suppress("SpreadOperator")
+            it.commandLine(*command)
+            it.standardOutput = outputStream
```

Comment:
will gradle close the streams automatically after? if not, to close them `.use` extension can be used.

anyway, not a big deal, because the streams passed are not using any external resources and just fake, so it won't affect anything.

Reply:
Yes they will be closed automatically